concepts/Common/Accessibility/00 Overview/15 Screen Reader Support.md

```diff
@@ -0,0 +1,7 @@
+Screen reader support allows users with visual impairments to access and comprehend the content. DevExtreme components include the following features that ensure proper screen reader work:
```

Comment:
```suggestion
Screen reader support allows users with visual impairments to access and comprehend the content. DevExtreme components include the following features that ensure the screen reader works properly:
```

Reply:
I'm not sure 'comprehend' is right here since using a screen reader does not guarantee you will understand the content